presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/health-checks/presto-health-check.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if test $# -gt 0; then
+    echo "$0 does not accept arguments" >&2
+    exit 32
+fi
+
+# Check if PrestoServer is listed as running Java process
+jps | grep -q PrestoServer
```

Comment:
run some SQL to check if it actually works?

Reply:
I don't think that it's possible in a generic way (Kerberos auth in Presto etc)